code/game/area/Space Station 13 areas.dm

```diff
@@ -2582,6 +2582,22 @@ area/security/podbay
 /area/awaycontent/a30
 	icon_state = "awaycontent30"
 
+////////////////////////VR AREAS///////////////////////////////////
+
+/area/vr
+	name = "VR"
+	requires_power = 0
+	dynamic_lighting = 0
+	no_teleportlocs = 1

```

Comment:
Though I think `no_teleportlocs` just means that effects like the wizard teleport and ghost "hop to area" can't go to these areas.